plugins/hangmanplugin.py

```diff
@@ -22,31 +22,35 @@ def startGame(self, ctx, msg):
 
         if msg.isWhisper:
             if len(args) < 1:
-                    ctx.msg("Not enough arguments. Please choose the game mode(Public/Private) and give me only one word for everyone to guess.", msg.player)
+                    ctx.msg("Not enough arguments. Please choose the game mode(Public/Private) and give me some word(s) for everyone to guess.", msg.player)
                     return
-            if len(args) > 2:
-                ctx.msg("Too many arguments. Please choose the game mode(Public/Private) and give me only one word for everyone to guess.", msg.player)
-                return
 
             if args[0].lower() == "private":
                 ctx.msg("You cannot play hangman solely from whisper... Yet.", msg.player)
                 return
             elif args[0].lower() == "public":
                 if len(args) < 2:
-                    ctx.msg("Not enough arguments. Please choose the game mode(Public/Private) and give me only one word for everyone to guess.", msg.player)
-                    return
-                if not args[1].isalpha():
-                    ctx.msg("The word you have told me contains characters other than letters. Please only use letters.", msg.player)
+                    ctx.msg("Not enough arguments. Please choose the game mode(Public/Private) and give me some word(s) for everyone to guess.", msg.player)
                     return
+                for arg in args[1:]:
+                    if not arg.isalpha():
+                        ctx.msg("The word you have told me contains characters other than letters. Please only use letters.", msg.player)
+                        return
 
                 if not self.word:
-                    self.word = args[1]
-                    # Make sure settings are back to beginning
+                    self.word = " ".join(args[1:])
                     self.guessedLetters = []
                     self.misses = 0
+                    # Make sure settings are back to beginning
                     ctx.msg("{:s} has given me a word. Try guessing some letters! (!guess <letter>)".format(msg.player))
-                    if not self.checkWord(ctx.config["hangman"]["api-key"], self.word):
-                        ctx.msg("This 'word' was not found in Wordnik!")
+                    badwords = 0
```

Comment:
This whole block of code that counts the number of "bad" words could actually be merged into the `for` loop above that's iterating over the `args` list. It's more efficient that way.